srml/support/src/lib.rs

```diff
@@ -31,10 +31,15 @@ extern crate srml_metadata;
 
 extern crate mashup;
 
+#[allow(unused_imports)]
+#[macro_use]
+extern crate srml_support_procedural;
+
 #[cfg(test)]
 #[macro_use]
 extern crate pretty_assertions;
 #[cfg(feature = "std")]
+#[allow(unused_imports)]
```

Comment:
Please remove this line, I know that the compiler is complaining. But this is probably an error in the nightly compiler. If the `macro_use` is really not needed, remove it.